docs/src/terraform/start-vm.md

```diff
@@ -0,0 +1,99 @@
+#### Running VMs
+
+Now it's time to prepare Terraform environment.
+
+```bash
+cd contrib/terraform/libvirt
+cp terraform.tfvars.json.example terraform.tfvars.json
+```
+
+After that, open the `terraform.tfvars.json` file with your favorite text
+editor. The only setting which you really need to change is
+`authorized_keys`. Please paste your public SSH key there. Otherwise,
+connecting to VMs with SSH will be impossible.
+
+Initialize the environment with:
+
+```bash
+terraform init
+```
+
+And then start the VMs:
+
+```bash
+terraform apply
+```
+
+ Terraform finished successfully, you should see the output with IP
+addresses of virtual machines, like:
+
+```bash
+ip_control_planes = {
+  "lockc-control-plane-0" = "10.16.0.225"
+}
```

Comment:
You need to close the code snippet here in this line, it doesn't render properly - https://mjura.github.io/lockc/terraform/start-vm.html

Reply:
Interesting that `mdbook test` didn't find this problem, during CI run

Reply:
Yes, looks like `mdbook test` is quite inefficient. Using something like https://github.com/markdownlint/markdownlint could be cool in the future.